src/store/tests/test-data/cat-b/__tests__/test-data.cat-b.selector.spec.ts

```diff
@@ -0,0 +1,385 @@
+import { QuestionOutcome } from '@dvsa/mes-test-schema/categories/common';
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
+import { CompetencyOutcome } from 'src/app/shared/models/competency-outcome';
+import { OutcomeBehaviourMapProvider } from 'src/app/providers/outcome-behaviour-map/outcome-behaviour-map';
+import { behaviourMap } from 'src/app/pages/office/office-behaviour-map';
+import { VehicleChecksQuestion } from 'src/app/providers/question/vehicle-checks-question.model';
+import {
+  hasSeriousFault,
+  getTestRequirements,
+  hasDangerousFault,
+  getETAFaultText,
+  getEcoFaultText,
+  getShowMeQuestionOptions,
+} from '../../common/test-data.selector';
+import {
+  getDrivingFaultCount,
+  getManoeuvres,
+  hasManoeuvreBeenCompletedCatB,
+  isTellMeQuestionSelected,
+  isTellMeQuestionCorrect,
+  isTellMeQuestionDrivingFault,
+  hasVehicleChecksBeenCompletedCatB,
+  hasEyesightTestGotSeriousFault,
+  hasEyesightTestBeenCompleted,
+} from '../test-data.cat-b.selector';
+import { Competencies } from '../../test-data.constants';
+import { cloneDeep } from 'lodash';
```

Comment:
think this should error with eslint due to import order, plus paths up top of file not using new aliasing

Reply:
linting should be fixed now